cdap-ui/app/cdap/components/PlusButtonModal/PlusButtonModal.less

```diff
@@ -75,6 +86,14 @@
 }
 
 .plus-button-modal {
+  .plus-modal-header-text {
```

Comment:
Talked to @PGuimarais we could use vertical-align here instead of position relative